source/security/index.markdown

```diff
@@ -1,11 +1,81 @@
 ---
 title: "Security"
-description: "Information about disclosing security vulnerabilities in Home Assistant."
+description: "Home Assistant takes its security seriously. This page contains information about how we handle security issues, how to report them, and information on past security issues."
 ---
 
-If you think that you have found a security vulnerability in Home Assistant, please disclose it to us via our security e-mail address at [security@home-assistant.io](mailto:security@home-assistant.io).
+Home Assistant takes its security seriously. We will do everything in our power to ensure that our users are safe.
 
-Please do not make vulnerabilities public without notifying us and giving us at least 3 days to respond.
+This page is intended to provide information about how we handle security issues and information about security issues we have [handled in the past](#past-advisories).
 
-If you are going to write about Home Assistant's security, please get in touch, so we can make sure that all claims are correct.
-  
+## Reporting a vulnerability
+
+So, you have found a security vulnerability in Home Assistant? Please, be sure to **responsibly disclose** it to us by [reporting a vulnerability using GitHub's Security Advisory](https://github.com/home-assistant/core/security/advisories/new).
```

Comment:
Presuming it's fine for reports about any HA component to be reported in the core repo then?

https://en.wikipedia.org/wiki/Coordinated_vulnerability_disclosure might be a good link for "responsibly disclose"

Reply:
> Presuming it's fine for reports about any HA component to be reported in the core repo then?

As the text listed, if you found something, use that link.